src/github.com/getlantern/flashlight/app/mock_autoupdate.go

```diff
@@ -0,0 +1,15 @@
+// +build mockupdate
+
+package app
+
+// This is a public key derived from a private key that is freely available.
+// Don't use in production.
+const packagePublicKey = `-----BEGIN PUBLIC KEY-----
```

Comment:
I opened an issue to find out why https://github.com/getlantern/lantern-pro/issues/119, this should be working on devel, tho.